Bigtable/src/ChunkFormatter.php

```diff
@@ -228,6 +228,9 @@ private function updateReadRowsRequest($request, $options, $prevRowKey)
     {
         if ($this->originalRowsLimit) {
             $request->setRowsLimit($this->originalRowsLimit - $this->numberOfRowsRead);
+            if ($this->numberOfRowsRead == $this->originalRowsLimit) {
```

Comment:
I believe this needs to be `===`, otherwise it'll set `requestCompleted` to true when `originalRowsLimit` is `null` and no rows have been read.

Reply:
Actually, this can't happen because we check that `originalRowsLimit` is truthy above, but it's still best practice to make it `===`